### PR TITLE
Support Yara Scan Flags

### PR DESCRIPTION
**Pull request by rc-grey**
Simply added support for Yara scan flags, exposed via the Rules object.

### CODE REVIEW

File: src/rules.rs [hunk 1]
```diff
@@ -7,15 +7,23 @@ use failure::ResultExt;
 
 use crate::{errors::*, initialize::InitializationToken, internals, YrString};
 
+pub const SCAN_FLAGS_FAST_MODE: i32 = 1;
+pub const SCAN_FLAGS_PROCESS_MEMORY: i32 = 2;
+pub const SCAN_FLAGS_NO_TRYCATCH: i32 = 4;
+pub const SCAN_FLAGS_REPORT_RULES_MATCHING: i32 = 8;
+pub const SCAN_FLAGS_REPORT_RULES_NOT_MATCHING: i32 = 16;
```
Comment (Hugal31) [inline src/rules.rs:14]: I'd prefer to import the flags with bindgen, and re-export them from yara_src.

**Review by Hugal31 — CHANGES_REQUESTED** [review]

**Comment by Hugal31** [comment]
It would also be great if you could add a test for your use case.

**Comment by rc-grey** [comment]
This may address your other issue regarding linking options, not sure.

**Review by Hugal31 — APPROVED** [review]

**Comment by rc-grey** [comment]
Do you mind making a release with this?

**Comment by Hugal31** [comment]
I don't know why Travis isn't showing up here but https://travis-ci.org/github/Hugal31/yara-rust/jobs/724992831

**Comment by rc-grey** [comment]
Sorry about that - fixed.

**Pull request merged**